client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FooterSection from 'calypso/components/footer-section';
+import LinkCard from 'calypso/components/link-card'; // should we host this one inside footer-section?
+
+interface Article {
+	title: string;
+	category: string;
+	url: string;
+	background: string;
+}
+
+const deslugify = ( string: string ) => string.replaceAll( '-', ' ' );
+const slugify = ( string: string ) =>
+	string
+		.replace( /[&/\\#,+()$~%.'":*?<>{}’]/g, '' )
+		.toLowerCase()
+		.replaceAll( ' ', '-' );
+
+const ArticleLinksContainer = styled.div`
+	@media ( max-width: 960px ) {
+		grid-template-columns: repeat( 1, 1fr );
+	}
+
+	@media ( max-width: 660px ) {
+		padding: 0 16px;
+	}
+
+	display: grid;
+	grid-template-columns: repeat( 3, 1fr );
+	column-gap: 29px;
+	row-gap: 10px;
+`;
+
+const EducationFooter = () => {
+	const translate = useTranslate();
```

Comment:
```suggestion
	import { useI18n } from '@wordpress/react-i18n';
	const { __ } = useI18n();
```
Ideally we should use core methods now they're supported.

Reply:
@lsl what is the difference between `useTranslate` and `useI18n` why we should favor one above the other?

Reply:
There shouldn't be much difference in output, the calypso version (`useTranslate`) predates core's translation methods (`useI18n`) which we couldn't use for quite a while due to support for [jsx interpolation](https://github.com/Automattic/wp-calypso/blob/7f44e630fdfb68cafdd12c0c1b5f25469ff9991e/client/signup/translations/seller-experience.js#L40-L44). Since core [added support](https://github.com/WordPress/gutenberg/issues/9846#issuecomment-555583946) for [`createInterpolateElement`](https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/) I don't think there is any reason to keep using/maintaining our own version.

@yuliyan / @p-jackson is there any reason to keep using calypso's translate methods? (Is community translator still a concern?) - asking because I've recommended to use `useI18n` over `useTranslate` a couple of time recently and now I'm wondering if I'm just making things up.

Reply:
The community translator still doesn't support `useI18n`, because there are a few limitations in @wordpress/i18n (i.e. `sprintf`, `createInterpolateElement`) that block us migrating it. However, since the community translator is considered an optional feature, using `useI18n` should be perfectly fine.

Reply:
TIL `useI18n()` doesn't support community translator! Good to know.

Reply:
Does @Automattic/i18n have a position on what we should use as the preferred method? 

I think there might be scenarios where `useI18n` makes more sense (like in packages where we don't want to pull in calypso code) but other than that maybe we should stick to the old `translate` method? How much is the community translator used? How likely is a future fix to support the core methods?

Reply:
@lsl, I believe we were recommending the use of `@wordpress/react-i18n` at some point, as we were planning migrating completely to it. However, due to having the blockers I've mentioned above and working on other higher priority projects, we've pushed it back on our roadmap.

From my personal observation, using either `i18n-calypso` or `@wordpress/react-i18n`, should be fine, but as a rule of thumb, I usually try to stick with whatever is already being used in the context (i.e. package, section, component, etc.) I'm working on.